### ISSUE TITLE
MA0011 should not be reported when all of the arguments are string

### ISSUE THREAD

**Issue by drieseng** [meziantou/Meziantou.Analyzer#533]
**MA0011** should not be reported when all of the arguments for `string.Format(...)` are of type **string**.

For example:

```c#
var x = string.Format("My name is, {0}? My name is {0}.", "who");
```

**Comment by meziantou** [comment]
Indeed, the rule should be smarter. It's not only for string values, but for any culture-insensitive type such as `char`.
